loader/app/db/session.py

```diff
@@ -0,0 +1,45 @@
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker, Session
+from typing import Generator
+
+import os
+
+engine = create_engine(
+    os.environ["DATABASE_URL"],
```

Comment:
Is this env var still defined?

Reply:
Yes, `LOADER_DATABASE_URL` is provided to the environment in docker-compose.yml like this:

```
    environment:
      DATABASE_URL: ${LOADER_DATABASE_URL}
```